tutorials/protocol.m

```diff
@@ -143,15 +155,27 @@
 
 %% Contrain with proteomics data
 % Load proteomics
-ecModelProt = readProteomics(ecModelTuned);
+protData = loadProtData(3); %Number of replicates
+ecModelProt = fillProtConcs(ecModelTuned,protData);
 ecModelProt = constrainProtConcs(ecModelProt);
-sol=solveLP(ecModelProt)
-[ecModelFlex, proteins, frequence] = flexibilizeProtConcs(ecModelProt);
 
-% The above fails, because some kcats are just very wrong. Manual curation
-% of the kcat values fixes this. Apply these custom kcats here:
-ecModelProt = applyCustomKcats(ecModelProt);
-ecModelProt = applyKcatConstraints(ecModelProt);
-
-% Now the model should predict growth at 0.4.
-sol=solveLP(ecModelProt)
+% Load matching flux data
+fluxData = loadFluxData();
+ecModelProtFlux = constrainFluxData(ecModelProt,fluxData);
+sol=solveLP(ecModelProtFlux)
+% Growth rate of 0.1 is by far not reached, flexibilize protein
+% concentrations
+[ecModelFlex, flexProt] = flexibilizeProtConcs(ecModelProtFlux,0.1,10);
+
+% It gets stuck at 0.0889. It seems like protein abundances are not
+% preventing the model to reach 0.1. First look if the conventional GEM is
+% able to reach 0.1 with the same constraints:
+modelY = constrainFluxData(modelY,fluxData);
+sol=solveLP(modelY)
+% It also only reaches 0.0889! So the metabolic network would not be able
+% to adhere to all measured constraints. Perhaps there is something
+% incorrect with the measurements? For now, we will limit the growth rate
+% to 0.08885 in our search:
+[ecModelFlex, flexProt] = flexibilizeProtConcs(ecModelProtFlux,0.08885,10);
```

Comment:
data in `customKcat.tsv` can be updated. I think the problem is with some complexes, like `Mitochondrial respiratory chain complex IV: r_0438 ` there are five rules here, but only have match with the data in `customKcats.tsv` other examples `Mitochondrial respiratory chain complex III: r_0439`, have 2 rules  `Mitochondrial isocitrate dehydrogenase complex (NAD+): `. same rules but complex data report 4 units. Then, kcat used in GECKO1&2 need to be multiplied ?

Reply:
Probably yes. Currently the protocol.m applies customKcats earlier, but it can also be done after a first attempt of running flexibilization. "Through simulations of numerous conditions, for ecYeastGEM various kcat values were manually curated, as now gathered in `customKcats.tsv`".